test/lib/cli.js

```diff
@@ -30,7 +30,7 @@ const cliMock = async (t, opts) => {
 }
 
 t.afterEach(() => {
-  delete process.exitCode
+  process.exitCode = undefined
```

Comment:
was this an intentional change in node? it may be worth filing an issue or commenting on the PR that made the change.

Reply:
you may be correct that this was unintentional. i found another breakage in related code. opened an issue here: https://github.com/nodejs/node/issues/45683